config.template/faf-aio-replayserver2/faf-aio-replayserver2.env

```diff
@@ -0,0 +1,15 @@
+LOG_LEVEL=20
+PORT=15000
+MYSQL_HOST=faf-db
+MYSQL_PORT=3306
+MYSQL_USER=faf-aio-replayserver
```

Comment:
Do you have created that user in `faf-db`?

Reply:
`create "faf" "faf-aio-replayserver" "${MYSQL_AIO_REPLAYSERVER_PASSWORD}"` in `faf-db` in `init/0_init_db_and_users.sh`